src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  padding: 16px 0 24px;
+  align-items: flex-end;
+  text-align: justify;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  &__star--active {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__review-count {
+    padding-left: 17px;
+    line-height: 14px;
+    text-align: right;
+    font-size: 10px;
+    color: $primary-color;
```

Comment:
The variable `$primary-color` is used here. Ensure that it is defined in your SCSS files, specifically in the `variables.scss` file, to avoid any compilation errors.